client/src/pages/ScanStore.tsx

```diff
@@ -49,39 +82,187 @@ function ScanStore() {
     setShoppingList(sList);
   };
 
+  const setDebugItem = (barcode: string) => {
+    setShoppingList(cartItems);
+    setCurBarcode(barcode);
+  };
+
+  const addItem = () => {
+    let barcodes: string[] = [curBarcode];
+    fetchItem(barcodes);
+  };
+
+  const fetchItem = async (barcodes: string[]) => {
+    let data = {
+      "merchant-id": merchantID,
+      barcode: barcodes,
+    };
+    fetchJson(data, "/api/items", displayItems);
+  };
+
+  const displayItems = (extractedItems: Item[]) => {
+    for (let i = 0; i < extractedItems.length; ++i) {
+      console.log(idxMap[extractedItems[i].barcode]);
```

Comment:
remove any console statements from check in code. They are fine while debugging.